server/cells.go

```diff
@@ -28,37 +28,31 @@ type dashboardCellResponse struct {
 
 func newCellResponse(dID chronograf.DashboardID, cell chronograf.DashboardCell) dashboardCellResponse {
 	base := "/chronograf/v1/dashboards"
-	newCell := chronograf.DashboardCell{}
-	newCell.Queries = make([]chronograf.DashboardQuery, len(cell.Queries))
-	copy(newCell.Queries, cell.Queries)
+	if cell.Queries == nil {
+		cell.Queries = make([]chronograf.DashboardQuery, 0)
```

Comment:
super nit-picky, but any reason for the preference of make to

```go
cell.Queries = []chronograf.DashboardQuery{}
```

Reply:
Fixed in 84b6cf4